client/dashboard/profile-wizard/index.js

```diff
@@ -90,32 +88,26 @@ class ProfileWizard extends Component {
 		return currentStep;
 	}
 
-	goToNextStep() {
+	async goToNextStep() {
+		const { addNotice, isError, updateProfileItems } = this.props;
 		const currentStep = this.getCurrentStep();
 		const currentStepIndex = getSteps().findIndex( s => s.key === currentStep.key );
 		const nextStep = getSteps()[ currentStepIndex + 1 ];
 
 		if ( 'undefined' === nextStep ) {
```

Comment:
I know it's not changed here, but this should be `'undefined' === typeof nextStep` to correctly work.